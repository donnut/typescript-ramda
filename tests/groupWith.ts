import R_groupWith = require('../ramda/dist/src/groupWith');

declare const string_to_boolean: (x: string) => boolean;
declare const string_array: string[];
declare const string: string;

// @dts-jest:pass:snap
R_groupWith(string_to_boolean);
// @dts-jest:pass:snap
R_groupWith(string_to_boolean, string);
// @dts-jest:pass:snap
R_groupWith(string_to_boolean, string_array);
