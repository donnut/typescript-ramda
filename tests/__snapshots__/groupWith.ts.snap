// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`R.groupWith(string_to_boolean) 1`] = `"<U extends string[] | ArrayLike<string>>(list: U) => U[]"`;

exports[`R.groupWith(string_to_boolean, string) 1`] = `"string[]"`;

exports[`R.groupWith(string_to_boolean, string_array) 1`] = `"string[][]"`;
